### ISSUE TITLE
Fix heading levels

### ISSUE THREAD

**Issue by bronzehedwick** [bronzehedwick/chrisdeluca#84]
All article summaries currently rely on the sectioning root algorithm, which no browser implements. The headings should be changed to reflect the real world.
